packages/synapse-react-client/src/components/SynapseChat/AccessLevelMenu.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
```

Comment:
Dropdown selector for the access level.  This is not approved by design, so will likely significantly change